stages/stage_0.py

```diff
@@ -9,7 +9,7 @@
 def run_program(
     program,
     args,
-    quote_kw=KEYWORD_TO_ATOM["q"],
+    quote_kw=KEYWORD_TO_ATOM["quote"],
     apply_kw=KEYWORD_TO_ATOM["a"],
```

Comment:
here too, it seems we're not following the naming convention of `_KW` being the *name* and `_ATOM` being the *value*/*op-code*